lib/crowbar/client/app/upgrade.rb

```diff
@@ -184,16 +184,23 @@ def admin
           catch_errors(e)
         end
 
-        desc "nodes",
-          "Trigger the upgrade on all nodes"
+        desc "nodes COMPONENT",
+          "Trigger the node upgrade"
```

Comment:
Could you please show the COMPONENTS here as well? (Just the values, it's enough space there)